src/container_registries.tf

```diff
@@ -0,0 +1,12 @@
+module "container_registries" {
+  source   = "./modules/container_registries"
```

Comment:
The folder name should be singular as the module alone will only create a single container registry.
The module invocation has the `for_each` so it's plural `module "container_registries"`